### ISSUE TITLE
Intermittent parallelized "make install" failure for 1.4.0

### ISSUE THREAD

**Issue by ilovezfs** [getdnsapi/getdns#382]
```
cd src/tools && /Applications/Xcode.app/Contents/Developer/usr/bin/make install-getdns_query
cd src/tools && /Applications/Xcode.app/Contents/Developer/usr/bin/make install-getdns_server_mon
/usr/bin/install -c -m 755 -d /usr/local/Cellar/getdns/1.4.0/bin
/usr/bin/install -c -m 755 -d /usr/local/Cellar/getdns/1.4.0/bin
install: mkdir /usr/local/Cellar/getdns/1.4.0/bin: File exists
make[1]: *** [install-getdns_server_mon] Error 71../../libtool --mode=install cp getdns_query /usr/local/Cellar/getdns/1.4.0/bin

make: *** [install-getdns_server_mon] Error 2
make: *** Waiting for unfinished jobs....
libtool: install: cp .libs/getdns_query /usr/local/Cellar/getdns/1.4.0/bin/getdns_query
```
This occurred on 1 out of 3 CI nodes during the 1.4.0 version bump in Homebrew/homebrew-core: https://github.com/Homebrew/homebrew-core/pull/24415

Full build log is here:
https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/19049/version=sierra/console

duplicate since that will eventually be deleted:
https://gist.github.com/ilovezfs/ddf667bd2448e41f8693e784856d183b

**Comment by wtoorop** [comment]
Acknowledged!  Trying to create the directory simultanously doesn't help.
I'll have a look...

**Comment by wtoorop** [comment]
Installing the `getdns_query` and `getdns_server_mon` tools in turn from the main `Makefile` should resolve this issue.  Patch attached:

* [0001-Bugfix-382-Do-not-install-tools-in-parallel.patch.gz](https://github.com/getdnsapi/getdns/files/1748405/0001-Bugfix-382-Do-not-install-tools-in-parallel.patch.gz)

**Comment by wtoorop** [comment]
Resolved in 1.4.1
